static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -58,6 +60,95 @@ function ActionSet({
   const canMarkReviewed =
     anySelected && (allInQuerySelected || selectedIssues.some(issue => !!issue?.inbox));
 
+  // Determine whether to nest "Merge" and "Mark as Reviewed" buttons insde
+  // the dropdown menu based on the current screen size
+  const theme = useTheme();
+  const nestMergeButton = useMedia(`(max-width: ${theme.breakpoints[2]})`);
+  const nestMarkReviewedButton = useMedia(`(max-width: ${theme.breakpoints[1]})`);
+
+  const menuItems: MenuItemProps[] = [
+    ...(nestMergeButton
```

Comment:
What might be nicer than this is having a key like `hidden` or something that just won't render the menu item, and we can just have `hidden: !nestMergeButton`

Reply:
good point! 👉https://github.com/getsentry/sentry/pull/31752/commits/bdd190a30624e3e258158e1a33ffef5e14eaebdb